### PR TITLE
Hotfix/ingress after upgrading

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Fix issue during any POST request in airflow:
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The referrer header is missing.</p>
```

more details from https://scotthelme.co.uk/a-new-security-header-referrer-policy/

### CODE REVIEW

File: charts/nginx/templates/nginx-headers-configmap.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ metadata:
 data:
   # Policy was generated using https://report-uri.com/home/generate
   Content-Security-Policy: "default-src 'self' *.{{ .Values.global.baseDomain }}; script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net cdn.astronomer.io cdn.metarouter.io cdn.segment.com www.google-analytics.com js.stripe.com widget.intercom.io js.intercomcdn.com cdn.lr-ingest.io; img-src 'self' data: *; connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} e.metarouter.io api.segment.com api.segment.io api-iam.intercom.io wss://nexus-websocket-a.intercom.io; style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net fonts.googleapis.com; frame-src js.stripe.com; font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com js.intercomcdn.com data:; worker-src blob:"
-  Referrer-Policy: "no-referrer"
+  Referrer-Policy: "no-referrer-when-downgrade"
```
Comment (andriisoldatenko) [inline charts/nginx/templates/nginx-headers-configmap.yaml:16]: >The browser will not send the referrer header when navigating from HTTPS to HTTP, but will always send the full URL in the referrer header when navigating from HTTP to any origin. It doesn't matter whether the source and destination are the same site or not, only the scheme.

**Comment by andriisoldatenko** [comment]
i think this bug fixed in ingress, e.g. we had `no-referrer`, but we still send referrer:
```git diff
2020/04/10 12:27:52 [debug] 177#177: *179 http proxy header:
"POST /devoid-impact-7504/airflow/connection/add HTTP/1.1
authorization: Bearer ***

Host: deployments.local.astronomer-development.com
X-Request-ID: fac5d08df98e2d11b45d994154d22751

X-Real-IP: 127.0.0.1
X-Forwarded-For: 127.0.0.1
X-Forwarded-Host: deployments.local.astronomer-development.com
X-Forwarded-Port: 443
X-Forwarded-Proto: https

X-Scheme: https
Content-Length: 2176
cache-control: max-age=0
origin: https://deployments.local.astronomer-development.com
upgrade-insecure-requests: 1

content-type: multipart/form-data; boundary=----WebKitFormBoundaryVaS6e9pgDSYSfMvR
user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36
sec-fetch-dest: document
accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
sec-fetch-site: same-origin
sec-fetch-mode: navigate
sec-fetch-user: ?1
- referer: https://deployments.local.astronomer-development.com/devoid-impact-7504/airflow/
+ referer: https://deployments.local.astronomer-development.com/devoid-impact-7504/airflow/connection/add
accept-encoding: gzip, deflate, br
accept-language: en,ru;q=0.9,en-US;q=0.8,en-GB;q=0.7,uk;q=0.6
cookie: session=****

"
```

File: charts/nginx/templates/nginx-headers-configmap.yaml (hunk 1 above)
Comment (ashb) [reply charts/nginx/templates/nginx-headers-configmap.yaml:16]: Is there a "same origin" type mode we can use here instead?

Comment (andriisoldatenko) [reply charts/nginx/templates/nginx-headers-configmap.yaml:16]: image

**Comment by andriisoldatenko** [comment]
related PR https://github.com/astronomer/ap-vendor/pull/17

File: charts/nginx/templates/nginx-headers-configmap.yaml (hunk 1 above)
Comment (ashb) [reply charts/nginx/templates/nginx-headers-configmap.yaml:16]: I think that would work?

Comment (andriisoldatenko) [reply charts/nginx/templates/nginx-headers-configmap.yaml]: @ashb done.
yeah tested locally, it works!

**Comment by andriisoldatenko** [comment]
@ashb PTAL

**Comment by schnie** [comment]
Typically we'd merge this to `master` then `cherry-pick` it out to the`release-0.13` branch. Any reason we didn't put this on `master`?

**Pull request merged**